src/typing/module_js.ml

```diff
@@ -263,37 +263,48 @@ module Node = struct
       match get_key "main" tokens with
       | None -> None
       | Some file ->
-          let path = Files_js.normalize_path dir file in
-          if path_is_file path
-          then Some path
-          else seq
-            (fun () ->
-              seqf
+	 let opts = match !flow_options with
```

Comment:
Great, actually I wanted to do that, but I was not sure if the "guidelines" allowed this.